src/wizard/Wizard.tsx

```diff
@@ -40,14 +38,8 @@ function Wizard({
   }
 
   return (
-    <Content
-      className="site-layout-background"
-      style={{
-        padding: 24,
-        position: "relative",
-      }}
-    >
```

Comment:
Sometime ago, I also played with removing the background on stepper and leave it only for step content, but for me, it was less attractive, so I reverted that. If we want to remove it anyway, maybe we can make the content background "cleaner"? Currently, somehow it reminds me of old apps

**Current**
![Screen Shot 2022-01-14 at 14 30 03](https://user-images.githubusercontent.com/17568639/149523092-6e1e4cc0-8ffd-4ca6-a8f8-9b1341b61a41.png)

**White background + changed border**
![Screen Shot 2022-01-14 at 14 30 27](https://user-images.githubusercontent.com/17568639/149523098-b8a7fd65-55bf-44cf-9c97-91e5551e3f66.png)
![2 3](https://user-images.githubusercontent.com/17568639/149523138-2434725b-3025-4268-82e9-4a0b72067628.png)

Reply:
I changed the styles, not only here, but also in other places 👍